src/pages/home/report/ReportActionsView.tsx

```diff
@@ -91,12 +92,9 @@ function ReportActionsView({
     const reportActionID = route?.params?.reportActionID;
     const prevReportActionID = usePrevious(reportActionID);
     const didLayout = useRef(false);
-    const didLoadOlderChats = useRef(false);
-    const didLoadNewerChats = useRef(false);
     const {isOffline} = useNetwork();
 
     const {shouldUseNarrowLayout} = useResponsiveLayout();
-    const contentListHeight = useRef(0);
```

Comment:
I have noticed that this ref was not used anywhere. It was updated in `onContentSizeChange` and never used after that or passed anywhere

Reply:
Kinda feel like ideally we would do that in a separate PR in case there is any regression with the hook